ql/instruments/zerocouponinflationswap.hpp

```diff
@@ -109,8 +109,8 @@ namespace QuantLib {
         //! "Payer" or "Receiver" refers to the inflation leg
         Type type() const { return type_; }
         Real nominal() const { return nominal_; }
-        Date startDate() const { return startDate_; }
-        Date maturityDate() const { return maturityDate_; }
+        virtual Date startDate() const override { return startDate_; }
+        virtual Date maturityDate() const override { return maturityDate_; }
```

Comment:
I think the idiomatic syntax is to add the `override` without the `virtual`.

Reply:
Okay, I have changed it.